mirna_explorer/repository/create_database.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import os
+import sqlite3
+
+data_folder = "../../../mirna_target_prediction/data/"
+
+required_files = ["raw/UTR_Sequences.txt",
+				  "processed/mirna_id_dictionary.tsv",
+				  "processed/gene_id_dictionary.tsv",
+				  "processed/scored_interactions_processed.tsv"]
```

Comment:
Weird formatting :D That's because there is a mix of spaces and tabs xD pycodestyle to the rescue!

1. install the `pycodestyle` module
1. run `pycodestyle create_database.py`.

Basically, the module checks the Python's code style outlined in PEP8 and outputs anything that is not conform to what the document has established. Spaces are used instead of tabs in Python :D

Reply:
There is a string inconsistency in the array. Use single quotes whenever possible

Reply:
Integrated `pycodestyle` which Emacs, we are not going to have PEP8 style errors anymore ;-)